### PR TITLE
135  view received application cards

### LINKED ISSUE #135: View received applications cards
**User Story:**
As a round operator
I want to see my received applications
So that I can decide what to do with them

**Acceptance Criteria**
GIVEN that I am on my round's homepage
WHEN the "Received" toggle is in the selected state
THEN I see the list of application project cards that have been submitted to my round

GIVEN that I am on my round's homepage
WHEN the "Received" toggle is in the unselected state
THEN I don't see the list of application project cards

**Product & Design Links:**
[Miro link](https://miro.com/app/board/uXjVOo2B--w=/?moveToWidget=3458764528805660545&cot=14)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by pragmatriot**
##### Description

Display received/pending applications for a round as cards

##### Refers/Fixes

[fixes #135]

##### Testing

Did a lot of work trying to figure out how to best pattern to stub data for the queries when testing components, but will break off a different story for getting a testing framework in place for future components.

preview

### CODE REVIEW

File: packages/round-manager/src/features/api/services/grantApplication.ts [hunk 1]
```diff
@@ -60,4 +60,5 @@ export const grantApplicationApi = api.injectEndpoints({
 
 export const {
   useListGrantApplicationsQuery,
-} = grantApplicationApi
\ No newline at end of file
+} = grantApplicationApi
+
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/api/services/grantApplication.ts:64]: we can remove the extra line here

File: packages/round-manager/src/test-utils.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    return (
+      <Provider store={ store }>
+        <ReduxRouter history={ history } store={ store }>
+          { children }
+        </ReduxRouter>
+      </Provider>
+    )
+  }
+
+  return rtlRender(ui,{wrapper: Wrapper, ...renderOptions})
```
Comment (thelostone-mc) [inline packages/round-manager/src/test-utils.tsx:27]:
```suggestion
  return rtlRender(ui ,{ wrapper: Wrapper, ...renderOptions })
```

File: packages/round-manager/src/mocks/handlers.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            },
+            title: "Some Title 2",
+            description: "Some project description 2",
+            payoutAddress: "0xSomeAddress2"
+          },
+          {
+            id: "3",
+            metaPtr: {
+              protocol: 1,
+              pointer: "some-ptr3"
```
Comment (thelostone-mc) [inline packages/round-manager/src/mocks/handlers.js:32]: would it be better to use actual pointer values ?

File: packages/round-manager/src/mocks/handlers.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            },
+            title: "Some Title 3",
+            description: "Some project description 3",
+            payoutAddress: "0xSomeAddress3"
```
Comment (thelostone-mc) [inline packages/round-manager/src/mocks/handlers.js:36]: would it be better to use actual eth address ?

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
Could you add a recording of the changes as well to the PR ?  
Also which round contract did we test this against ? I was checking out the graph query and test it out myself as well

File: packages/round-manager/src/mocks/handlers.js (hunk 3 above)
Comment (pragmatriot) [reply packages/round-manager/src/mocks/handlers.js:32]: Thinking so, but it should all be mocked by MSW anyways. We put a story in the backlog to setup a better testing framework around rendering context-aware components. I'll make this something real for now, though.

File: packages/round-manager/src/mocks/handlers.js (hunk 4 above)
Comment (pragmatriot) [reply packages/round-manager/src/mocks/handlers.js:36]: Same as other MSW comment, but will swap it for a real value for now

File: packages/round-manager/src/features/api/services/grantApplication.ts (hunk 1 above)
Comment (pragmatriot) [reply packages/round-manager/src/features/api/services/grantApplication.ts:64]: Sure. :) Force of habit. 😊

**Review by thelostone-mc — APPROVED** [review]

**Comment by thelostone-mc** [comment]
Oh looks like we have round-manager test failing ! Wanna fix that up + add a screenshot of the UI so that @nfrgosselin can check it as well ?

**Comment by chibie** [comment]
> Oh looks like we have round-manager test failing ! @pragmatriot Wanna fix that up + add a screenshot of the UI so that @nfrgosselin can check it as well ?

@pragmatriot i suggest you remove the component test files from this pr so the build can pass + move them to a new branch

**Review by chibie — CHANGES_REQUESTED** [review]
can you remove the component test files from this pr so the build can pass + move them to a new branch so the build can pass?

**Pull request merged**